docs/source/release/v6.0.0/framework.rst

```diff
@@ -5,61 +5,56 @@ Framework Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
+.. figure:: ../../../../images/mantid_workbenchnightly.png
+   :class: screenshot
+   :width: 200px
+   :align: right
 
 Installation
 ------------
 
-- The macOS bundle is now suffixed with ``Nightly`` if it comes from a nightly development build and can be installed alongside a full release build.
+- **The macOS bundle is now suffixed with** ``Nightly`` **if it comes from a nightly development build and so can be installed alongside a full release build.**
 
 Concepts
 --------
 
-- Change sphinx documentation to use mathjax to render equations
-- Added packing fraction to :ref:`Material <Materials>` to separate number density and effective number density.
-- Added a feature allowing time-dependent values for individual instrument parameters.
+- Sphinx documentation now uses MathJax to render equations
+- Time-dependent values for individual instrument parameters are now allowed
 - Added a ``PythonStdOutChannel`` and the ability to change the :ref:`logging channel <Properties File>` without restarting mantid framework
+- Added packing fraction to :ref:`Material <Materials>` for separate and effective number density
 
 Algorithms
 ----------
 
-- :ref:`CalculateEfficiency <algm-CalculateEfficiency>` has now overwritten `processGroups` method and  a new property, MergeGroups, that is used to flag that the input group
-  should be merged and averaged, and masked spectra be filled with data (if available) from other entries of the input group instead of processing them entry-by-entry.
-- The calculation of a distance has been updated in Track to correctly calculate the distance for objects that have multiple intercepting surfaces, e.g. hollow cylinder. This affect algorithms such as :ref:`AbsorptionCorrection <algm-AbsorptionCorrection>` where you may now get slightly different values.
-- Added the ability to specify the packing fraction and effective number density to :ref:`SetSample <algm-SetSample>` and :ref:`SetSampleMaterial <algm-SetSampleMaterial>`.
+- :ref:`SetSample <algm-SetSample>` and :ref:`SetSampleMaterial <algm-SetSampleMaterial>` allow setting the packing fraction and effective number density
+- The calculation of distance is now correct for tracks passing through objects have multiple intercepting surfaces, e.g. hollow cylinder. This affects algorithms such as :ref:`AbsorptionCorrection <algm-AbsorptionCorrection>`.
```

Comment:
This is a bit pedantic but I see there's a merge conflict that will need fixing so you'll need to do one more change anyway.
Please can you add "that" ie "....for tracks passing through objects that have multiple..."